ytlister/settings.py

```diff
@@ -20,10 +27,10 @@
 # See https://docs.djangoproject.com/en/3.1/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = '@b13yji#z=m3yh)gq5!_vw9d1u7y)lu&2=*txdu)7qfw%vo^3='
```

Comment:
SECRET moved from this file to .env file for security. Else anyone can hijack the session of this web app